code/modules/mob/living/simple_animal/hostile/ordeal/green.dm

```diff
@@ -141,9 +161,9 @@
 		if(!istype(target, /mob/living))
 			return
 		var/turf/T = get_turf(target)
+		if(!T)
+			return
 		for(var/i = 1 to 4)
-			if(!T)
-				return
 			new /obj/effect/temp_visual/saw_effect(T)
```

Comment:
Although unlikely, the turf check was in loop because it *could* be deleted in-between attacks.